app/controllers/api/databases.php

```diff
@@ -2398,14 +2403,19 @@ function updateAttribute(
     ->param('collectionId', '', new UID(), 'Collection ID. You can create a new collection using the Database service [server integration](https://appwrite.io/docs/server/databases#databasesCreateCollection).')
     ->param('key', '', new Key(), 'Attribute Key.')
     ->param('required', null, new Boolean(), 'Is attribute required?')
-    ->param('min', null, new FloatValidator(), 'Minimum value to enforce on new documents')
-    ->param('max', null, new FloatValidator(), 'Maximum value to enforce on new documents')
+    ->param('min', null, new FloatValidator(), 'Minimum value to enforce on new documents', true)
+    ->param('max', null, new FloatValidator(), 'Maximum value to enforce on new documents', true)
     ->param('default', null, new Nullable(new FloatValidator()), 'Default value for attribute when not provided. Cannot be set when attribute is required.')
     ->param('newKey', null, new Key(), 'New attribute key.', true)
     ->inject('response')
     ->inject('dbForProject')
     ->inject('queueForEvents')
     ->action(function (string $databaseId, string $collectionId, string $key, ?bool $required, ?float $min, ?float $max, ?float $default, ?string $newKey, Response $response, Database $dbForProject, Event $queueForEvents) {
+
+        // Ensure attribute default is within range
+        $min = \is_null($min) ? -PHP_FLOAT_MAX : $min;
```

Comment:
```suggestion
        $min ??= PHP_INT_MIN;
```

Reply:
is there no float_min? 
if not I think the minimum value would be `-PHP_FLOAT_MAX + 1` 🤔